src/Routes/OrderDetails/OrderDetails.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+import classes from "./OrderDetails.module.css";
+
+const OrderDetails = () => {
+  const params = useParams();
+  const orders = useSelector((state) => state.order.orders);
+  const currentOrderName = orders.find(
+    (order) => order.orderName === params.orderName
+  );
+
+  return (
+    <>
+      {orders.length !== 0 && currentOrderName ? (
+        <div className={classes["order-data-info"]}>
+          <div>
+            <div className={classes["info-container"]}>
+              <h2>Delivery Information</h2>
+              <div>
+                <strong>Name: </strong>
+                <span>{`${currentOrderName.deliveryAdress.firstName} ${currentOrderName.deliveryAdress.lastName}`}</span>
+              </div>
+              <div>
+                <strong>Delivery adress: </strong>
+                <span>{`${currentOrderName.deliveryAdress.city}, ${currentOrderName.deliveryAdress.street}`}</span>
+              </div>
+              <div>
+                <strong>Date of order: </strong>
+                <span>{currentOrderName.orderDate}</span>
+              </div>
+              <div>
+                <strong>Delivery status: </strong>
+                <span>{currentOrderName.status}</span>
+              </div>
+            </div>
+            <div className={classes["info-container"]}>
+              <h2>Products</h2>
+              <ul>
+                {currentOrderName.orderedItems.map((el) => (
```

Comment:
Please destructure the properties from this object
```suggestion
                {currentOrderName.orderedItems.map(({id,size}) => (
```
And mentioning the id in the key arg will be enough

Reply:
If I have in the order same sneakers but the size is different or the amount is same, for each sneakers will repeat the id. I need 2 args that differentiate the id